### PR TITLE
add fix in consensus method to run on python 3.x

### PR DESCRIPTION
**Pull request by Gasta88**
This pull request addresses PR #1386

This is a fix for the method `consensus()` in `Bio.Nexus.Tree` to run in Python 3.

### CODE REVIEW

**Comment by Gasta88** [comment]
Appveyor crashed on `test_GraphicsGeneral.py`, I'll update the PR later this day.

File: Bio/Nexus/Trees.py [hunk 1]
```diff
@@ -40,9 +40,7 @@ def __init__(self, taxon=None, branchlength=0.0, support=None, comment=None):
 
 
 class Tree(Nodes.Chain):
-    """Represents a tree using a chain of nodes with on predecessor (=ancestor)
-    and multiple successors (=subclades).
-    """
+    """Represent a tree using a chain of nodes with on predecessor (=ancestor) and multiple successors (=subclades)."""
```
Comment (peterjc) [inline Bio/Nexus/Trees.py]: Its good that is now a one line summary, but its rather long. Can you perhaps split this into a one line summary, blank line, and a longer line?

**Comment by peterjc** [comment]
The work looks good, and includes some docstring updates too.

I've rerun the AppVeyor test which broke on ``test_simple_scatter_plot_1`` which happens occasionally as it uses a random set of points. Ideally we'd hard code that with a fixes list - or perhaps run the random number generator with a fixed seed?

**Comment by peterjc** [comment]
I'm interpreting Eric's comment https://github.com/biopython/biopython/pull/1386#issuecomment-327854926 as agreement with the proposed sorting fix.

**Comment by peterjc** [comment]
Before merging, I should have asked you to include your new tests for the ``merge_with_support`` and ``consensus`` (mentioned on #1386), but could you do those as a new pull request? Thanks!

**Pull request merged**